packages/calcite-components/src/components/navigation-logo/navigation-logo.scss

```diff
@@ -1,3 +1,16 @@
+/*
+* These properties can be overridden using the component's tag as selector.
+*
+* @prop --calcite-navigation-logo-background-color: Specifies the background color of the component.
+* @prop --calcite-navigation-logo-background-color-hover: Specifies the background color of the component when hovered.
+* @prop --calcite-navigation-logo-background-color-press: Specifies the background color of the component when active.
+* @prop --calcite-navigation-logo-border-color: Specifies the border color of the component.
+* @prop --calcite-navigation-logo-description-text-color: Specifies text color of description of component.
+* @prop --calcite-navigation-logo-heading-text-color: Specifies the text color of heading of component.
+* @prop --calcite-navigation-logo-icon-color-press: Specifies the icon color of the component when active.
```

Comment:
should this be `--calcite-navigation-logo-color-press`? The reason for using `--calcite-navigation-logo-icon-color-press` is to indicate the color only affects the icon since there is no text at host level other than `heading` or `description` which has it own tokens.

Reply:
the `heading - description` pattern should be 

`--calcite-[component]-text-color: Specifies the component's default color for description and icons`

`--calcite-[component]-heading-text-color: Specifies the component's heading text color.`